### ISSUE TITLE
Removing authorization deletes all local notes

### ISSUE THREAD

**Issue by dolfandringa** [nvbn/everpad#252]
First of all I have to say I am very happy with everpad. It is a great tool, that finally (since evernote never gave linux any love) provides all the required evernote functionality in an easy to use, fully desktop integrated fashion.
And now on to the ranting (it's not meant like that, sorry if it sounds like it) :)

I had a problem where synchronization didn't work anymore (something about Tag.name call throwing a NonetypeObject has no attribute len() exception). I read on another issue that removing the authorization and re-authorizing might solve the problem (which it indeed did). What both that ticket, and the interface of everpad fail to mention though, is that removing the authorization also deletes all local notes which haven't been synced yet. So I ended up loosing the meeting notes that I had made, but which hadn't been successfully synced yet. 
At least a warning would be nice when removing the authorization. It would be even better if a backup of the local database would be made when removing authorization, so later on you could merge the two somehow, or at least copy paste the contents of old notes into new ones. That would have saved me re-doing 45 minutes of work :)
Regards,

Dolf.

**Comment by hdave** [comment]
The "ask" solution is nice...but I just ran into this because of an authorization problem.  I reauthorized and lost several notes because of it.  I was warned (which was HUGE) so I made backup copies.  But if there could be an option to preserve new, never before synced notes that would be great.

**Comment by wadechandler** [comment]
Yes, apparently where evernote had their security issue, I had to reauthorize which meant I had to remove it and add it back; asking didn't change the fact that to keep using it I had to remove the authorization. I noticed everpad-provider was getting errors on the password when I ran it with --verbose. Anyways, reauthorized, and it is working again, and btw THANK YOU...because everpad is AWESOME...but I did have to back those up and then re-add them...the ones I had changed since the last sync, and really I guess the worst part was there was nothing to tell me that syncing had stopped nor why. I only noticed when I realized the last sync was hundreds of hours ago. All and all I'm up and running again, but this would be a very nice feature.

So I imagine the cases where the names don't match, the notes could easily be kept. Better to have some notes get created that may have been removed and delete them if needed versus possibly losing data. Next case, if notes have the same name, you could ask which one should take precedence, and let the user choose. I had a couple I had changed locally, but the name remained the same, and had to copy those back in manually.

**Comment by nathvarun** [comment]
HELP.i think removing authorization just removed notes that i had been making for over a month! . This is weird as everpad showed that it used to sync regularly ( indicated by the last sync time) . 
I have reauthorized everpad but my notes are missing! 

i dont think this is how its meant to work because it showed it was syncing regularly. Any help would be appreciated 

edit : Does everpad store store the notes that on my system ? if they havent been synced then they must be somewhere on my system ? Im using ubuntu,
